app/controllers/v0/letters_controller.rb

```diff
@@ -29,7 +29,7 @@ def beneficiary
     private
 
     def service
-      @service ||= EVSS::Letters::ServiceFactory.get_service(mock_service: Settings.evss.mock_letters)
+      @service ||= EVSS::Letters::Service.new
```

Comment:
Think we can lose the memoization here, guilty of over memoizing things myself, in this case every request makes a new controller instance so it's not needed.